src/components/DonationItemsTable/DonationItemsTable.jsx

```diff
@@ -12,76 +22,238 @@ import {
   Tab,
   TabPanels,
   TabPanel,
+  Box,
+  IconButton,
+  Image,
+  Button,
+  Text,
+  useDisclosure,
 } from '@chakra-ui/react';
-import { DeleteIcon, EditIcon } from '@chakra-ui/icons';
-import PropTypes from 'prop-types';
+import NotificationsDrawer from '../NotificationsDrawer/NotificationsDrawer.jsx';
 
 const DonationItemsTable = () => {
+  const { isAdmin } = useAuth();
+  const [isAdminUser, setIsAdminUser] = useState(false);
+  const [notification, setNotification] = useState([]);
+  const { backend } = useBackend();
+  const navigate = useNavigate();
+  useEffect(() => {
+    const checkIsAdmin = async () => {
+      if (!(await isAdmin())) {
+        navigate('/BusinessDashboard');
+      } else {
+        setIsAdminUser(true);
+      }
+    };
+
+    const fetchNotifications = async () => {
+      const response = await backend.get('/notification/0');
+      setNotification(response.data);
+    }
+    checkIsAdmin();
+    fetchNotifications();
+  }, [backend]);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has missing dependencies: 'isAdmin' and 'navigate'. Either include them or remove the dependency array.